src/app/pages/loading-waiting-states/progress-bar-indeterminate/progress-bar-indeterminate.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-toolbar data-cy="pxb-toolbar" flexLayout="row" class="app-bar mat-elevation-z2" [color]="checked ? 'primary' : ''">
+    <button data-cy="toolbar-menu" *ngIf="isSmall" mat-icon-button (click)="toggleMenu()" style="margin-left: -8px">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Progress Bars (Indeterminate)</div>
+    <mat-progress-bar mode="indeterminate"></mat-progress-bar>
+    <pxb-spacer></pxb-spacer>
+    <mat-slide-toggle labelPosition="before" [checked]="checked" (change)="onToggle($event)">Use Dark Theme</mat-slide-toggle>
+</mat-toolbar>
+<section class="mat-typography" [class.small]="isSmall">
```

Comment:
Page body shall be white200

Reply:
this should be a global style somewhere for the app shouldn't it? Shouldn't this come from our theme?

Reply:
I couldn't find global style for it.

Reply:
@emclaug2 will know

Reply:
nvm, I see he already commented below